projects/dex-ui/src/components/Liquidity/RemoveLiquidity.tsx

```diff
@@ -0,0 +1,320 @@
+import React, { useCallback, useEffect, useMemo, useState } from "react";
+import { TokenInput } from "src/components/Swap/TokenInput";
+import { Token, TokenValue } from "@beanstalk/sdk";
+import { useAllTokensBalance } from "src/tokens/useTokenBalance";
+import styled from "styled-components";
+import { images } from "src/assets/images/tokens";
+import { useAccount } from "wagmi";
+import { ContractReceipt } from "ethers";
+import { Well } from "@beanstalk/sdk/Wells";
+import { useQuery } from "@tanstack/react-query";
+
+type RemoveLiquidityProps = {
+  well: Well;
+};
+
+enum REMOVE_LIQUIDITY_MODE {
+  Balanced,
+  OneToken,
+  Custom
+}
+
+type LiquidityAmounts = {
+  [key: number]: TokenValue;
+};
+
+export const RemoveLiquidity = ({ well }: RemoveLiquidityProps) => {
```

Comment:
I've been using this type for all react functional components, it adds some built in react things to the props, like `children`, tho not needed in this case:

```
import { FC } from "src/types";
export const RemoveLiquidity = FC<RemoveLiquidityProps>({ well }) => {
```